test/distributed/org/apache/cassandra/distributed/test/TestBaseImpl.java

```diff
@@ -43,4 +51,29 @@ public static void beforeClass() throws Throwable {
         // fact that this code is going to work accross _all_ versions.
         return Cluster.build();
     }
+
+    public static Object list(Object...values)
+    {
+        return CQLTester.list(values);
+    }
+
+    public static Object set(Object...values)
+    {
+        return CQLTester.set(values);
+    }
+
+    public static Object map(Object...values)
+    {
+        return CQLTester.map(values);
+    }
+
+    public static ByteBuffer tuple(Object... values)
+    {
+        return CQLTester.tuple(values).toByteBuffer();
+    }
+
+    public static ByteBuffer udt(Object... values)
+    {
+        return CQLTester.userType(values).toByteBuffer();
+    }
```

Comment:
@adelapena One thing we have to watch out for is that the long term plan for in-JVM dtests might involve them being in a separate repository. If that happens, the dependency on `CQLTester` might not be possible. For now, perhaps it would be best to just import the static methods from `CQLTester` and leave this uncluttered (and deal with that problem when we have to).

CC @ifesdjeen @dcapwell

Reply:
for list, set, and map I would be in favor of removing in favor of `Arrays.asList`, `Sets.newHashSet`, `ImmutableMap.of` as they are type safe and do the same thing; tuple and UDTs are harder to work with so offer more value (namely `org.apache.cassandra.cql3.CQLTester#typeFor` to infer the AbstractType recursively to generate the bb), given this PR though, tuple and UDTs are dead code so maybe we remove until needed?

Reply:
+1 for `list()` and `set()`, although `map()` looks slightly more involved.

Reply:
Sorry, you're right RE `map()`

Reply:
I'm afraid `ImmutableMap.of` can't be used since it doesn't admit null values like the ones used by the test. I'm instead just copying the `CQLTester#map` method, which is quite small. 

As for UDTs, the only difference with tuples is in the `toCQLString` implementation that we don't use, so they can be replaced by tuples. And for tuples, I have added in `TestBaseImpl` a simplified implementation of `CQLTester#tuple` that only supports not-complex types, so no nested tuples, UDTs not collections are allowed. I think that should be enough for the tests at hand. If we need more complex tuples in the future we should either just fully duplicate `CQLTester` methods or put them in some other place. WDYT?